Common/Field/src/MagFieldContFact.cxx

```diff
@@ -27,7 +27,7 @@ void MagFieldContFact::setAllContainers()
 {
   //  Creates the Container objects and adds it to the list of containers
   
-  FairContainer* p = new FairContainer("MagFieldParam", "Mag. Field Parameters", "Default Field");
+  auto* p = new FairContainer("MagFieldParam", "Mag. Field Parameters", "Default Field");
```

Comment:
You know the type, at least in this case, right? You have `new FairContainer` next to it. Do you see any cases in which this is not the case? If yes, I agree we probably do not want to do it globally.